cdc/redo/manager.go

```diff
@@ -391,15 +391,30 @@ func (m *ManagerImpl) flushLog(
 	go func() {
 		defer atomic.StoreInt64(&m.flushing, 0)
 
-		err := m.writer.FlushLog(ctx, tableRtsMap)
+		var minResolvedTs uint64 = 0
+		if len(tableRtsMap) > 0 {
+			minResolvedTs = math.MaxUint64
+			for _, resolvedTs := range tableRtsMap {
```

Comment:
Consider the following case:
1. current state: `m.rtsMap` manage two tables: t1, t2. And `m.rtsMap[t1] = 1, m.rtsMap[t2] = 2`, `m.minResolvedTs = 1`.
2. `tableRtsMap = {"t2": 3}`
 Then, `minResolvedTs` computed here will be 3， which is too large.

Maybe, we should calculate `minResolvedTs` in `bgUpdateLog`.